### DISCUSSION TITLE
option to hide "background tasks are running" circle

### DISCUSSION THREAD

**Discussion by jgbar** [music-assistant/support#893] [Feature requests and ideas]
I'd like a option to hide the background task circle or change it to a static icon instead of a rotating circle. 

I noticed users easily misinterpreted the circle as "system is loading, please wait" notification..

**Comment by DutchJaFO** [comment]
It is kind of a multi-use icon at the moment :

1. There is a real "system is loading" moment when the database is created and filled for the very first time.
2. The second situation (and the one you are likely to see) is when the system is merely updating specific bits of the database ("adding album/artist/track to database tasks"). You will see this when you navigate to a specific artist/album/track for the very first time.
3. The 'analyzing audio' audio tasks. I don't know what triggers them, but I'm seeing them when there are (new?) spotify tracks playing in my queue 

All of these are technically "the system is loading stuff ... please wait" kind of states, but I agree that with the exception of the first use case the icon can confuse the (non technical) users.

I'd also would like to add that the pop-up display and the information displayed is a bit too technical for folk without access to the internal database. Things like "adding spotify artist [magic number]" doesn't mean a thing to me, but "adding/updating [artist name/album name] does. I suspect this may be a technical limitation for data from streaming services, but it would help a lot if these notifications were a bit more human-friendly.

Moving that information to a separate page in the UI and maybe adding some kind of 'severity' system for these notifications similar to the logging within HA itself could help reduce the information overload during normal use as well.

Maybe a configuration setting that allows users to decide how much of the task-detail notifications they want to see ?
